index.html

```diff
@@ -0,0 +1,1376 @@
+                            <a href="#">
+                                <img src="assets/more-stories-8.jpg" alt="opinion" class="more-stories-img pr-3"></a>
+                            <div class="p-1 bd-highlight">
+                                <h6 class="font-weight-bold section-heading">
+                                    <a href="#" class="feature-hover">Coca-Cola and PepsiCo Ditch Ties to Plastics
+                                        Industry
+                                        Association</a>
+                                </h6>
+                                <p class="mt-1 more-stories-p"> The injunction is seen as a victory to opponents of the
+                                    laws,
+                                    who argued that they were not medically necessary and violated Supreme Court
+                                    precedent.
+                                </p>
+                            </div>
+                        </article>
+                    </section>
+                    <!--end of More stories section-->
+
+                </section>
+                <!--END OF TOP STORY SECTION-->
+                <!--START OF OPINION SECTION-->
+                <section
+                    class="col-xl-3 col-lg-3 col-md-12 col-sm-12 mt-4 m-0 p-0 order-sm-third ml-sm-4 ml-md-0 order-md-third pr-4l">
+                    <h6 class="font-weight-bold section-heading">
+                        OPINION
+                    </h6>
+
+                    <article class="d-flex bd-highlight border-bottom border-muted">
+                        <a href="#">
+                            <img src="assets/opinion-1.png" alt="opinion" class="img-fluid"></a>
+                        <div class="p-1 bd-highlight">
+                            <span class="font-weight-bold opinion-main-headline float-right">
+                                <a href="#">The Mueller Report's Biggest Open Secret</a>
+                            </span>
+                            <p class="mt-4 opinion-sub-headline font-weight-bold">BY SETH ABRAMSON</p>
+                        </div>
+                    </article>
+
+                    <article class="d-flex pt-2 bd-highlight border-bottom border-muted opinion-cover">
+                        <a href="#">
+                            <img src="assets/opinion-2.png" alt="opinion" class="img-fluid"></a>
+                        <div class="p-1 bd-highlight">
+                            <span class="font-weight-bold opinion-main-headline float-right">
+                                <a href="#">The Mueller Report's Biggest Open Secret</a>
+                            </span>
+                            <p class="mt-4 opinion-sub-headline font-weight-bold">BY SETH ABRAMSON</p>
+                        </div>
+                    </article>
+
+                    <article class="d-flex pt-3 bd-highlight border-bottom border-muted">
+                        <a href="#">
```

Comment:
LATEST NEWS is wrapped by `<section>`, and each article has `<article>` tag around.